wagtail/wagtailadmin/utils.py

```diff
@@ -194,12 +198,22 @@ def send_notification(page_revision_id, notification, excluded_user_id):
     }
 
     # Send emails
+    sent_count = 0
     for recipient in email_recipients:
-        # update context with this recipient
-        context["user"] = recipient
-
-        # Get email subject and content
-        email_subject, email_content = render_to_string(template, context).split('\n', 1)
+        try:
+            # update context with this recipient
+            context["user"] = recipient
+
+            # Get email subject and content
+            email_subject, email_content = render_to_string(template, context).split('\n', 1)
+
+            # Send email
+            send_mail(email_subject, email_content, [recipient.email])
+            sent_count += 1
+        except Exception as e:
+            logger.error(
```

Comment:
We could use [``logger.exception``](https://docs.python.org/2/library/logging.html#logging.Logger.exception) here. This would make useful debugging information (eg, a traceback) available to the log handler.